hs_communities/management/commands/check_czo_groups.py

```diff
@@ -0,0 +1,208 @@
+                                                                                 r.title.encode(
+                                                                                     'ascii',
+                                                                                     'ignore'
+                                                                                 ),
+                                                                                 prefix))
+                        # Fix it NOW
+                        GroupResourcePrivilege.unshare(resource=r, group=group, grantor=national)
+                    owners = User.objects.filter(u2urp__resource=r,
+                                                 u2urp__privilege=PrivilegeCodes.OWNER)
+                    if user not in owners:
+                        print("  User {} does not own resource".format(username))
+                        for o in owners:
+                            print("    {}".format(o.username))
+                    if national not in owners:
+                        print("  CZO national user does not own resource {}".format(r.short_id))
+                        UserResourcePrivilege.share(resource=r, user=national,
+                                                    privilege=PrivilegeCodes.OWNER,
+                                                    grantor=national)
+                    owners = User.objects.filter(u2urp__resource=r,
+                                                 u2urp__privilege=PrivilegeCodes.OWNER)
+                    if owners.count() == 0:
+                        print("  No owners for {} {}:".format(r.short_id,
+                                                              r.title.encode('ascii', 'ignore')))
+                        for o in owners:
+                            print("    {}".format(o.username))
+                    elif owners.count() == 1:
+                        print("  Too few owners for {} {}:".format(r.short_id,
+                                                                   r.title.encode('ascii',
+                                                                                  'ignore')))
+                        for o in owners:
+                            print("    {}".format(o.username))
+
+                    elif owners.count() == 2:
+                        pass  # print("  Proper number of owners")
+
+                    elif owners.count() > 2:
+                        print("  Extra owners for {} {}:".format(r.short_id,
+                                                                 r.title.encode('ascii', 'ignore')))
+                        for o in owners:
+                            print("    {}".format(o.username))
+
+                    groups = Group.objects.filter(g2grp__resource=r)
+
+                    if group not in groups:
+                        print("   Resource not in group {}".format(groupname))
```

Comment:
Should probably include the resource with this statement?

Reply:
Done in all cases.